[bug] Firebase Emulator Action Failure

# Bug Report: Firebase Emulator Action Failure

## Description
The Firebase Emulator GitHub Action is failing during the setup phase, specifically when trying to execute the [start-firebase-emulator.sh](https://github.com/invertase/firebase-emulator-action/blob/main/start-firebase-emulator.sh) script.

## Environment
- Repository: https://github.com/HassanBahati/firebase-emulator-action-example
- GitHub Action: [run-emulator](https://github.com/HassanBahati/firebase-emulator-action-example/actions/runs/11873147545/job/33087665607)
- Action Version: invertase/firebase-emulator-action@v1.0.0
- Workflow file: [firebase-emulator.yml](https://github.com/HassanBahati/firebase-emulator-action-example/blob/main/.github/workflows/firebase-emulator.yml)

## Steps to Reproduce
1. Attempts to start emulators with;
```sh
              - name: Firebase Emulator Action
                uses: invertase/firebase-emulator-action@v1.0.0
            
```

## Current Behavior
- Action fails with error: `chmod: cannot access './start-firebase-emulator.sh': No such file or directory`
- Process exits with code 1

## Expected Behavior
The start-firebase-emulator.sh script should be present and executable, allowing the Firebase emulator suite to start successfully.

## Logs
```log
Run invertase/firebase-emulator-action@v1.0.0
Run sudo npm i -g firebase-tools@latest
added 643 packages in 18s
69 packages are looking for funding
run `npm fund` for details
Run chmod +x ./start-firebase-emulator.sh && ./start-firebase-emulator.sh auth,firestore,functions,storage,database test-project 3 60 1 8080
chmod: cannot access './start-firebase-emulator.sh': No such file or directory
Error: Process completed with exit code 1.
```
## ScreenShot
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/df94d9f3-abb0-4790-9fcd-356d39494329">

--- a/action.yaml
+++ b/action.yaml
@@ -43,4 +43,11 @@ runs:
     - name: Start Firebase Emulator
       shell: bash
       run: |
-       chmod +x ./start-firebase-emulator.sh && ./start-firebase-emulator.sh ${{ inputs.emulators }} ${{ inputs.project-id }} ${{ inputs.max-retries }} ${{ inputs.max-checks }} ${{ inputs.wait-time }} ${{ inputs.check-port }}
+        chmod +x ${{ github.action_path }}/start-firebase-emulator.sh
+        ${{ github.action_path }}/start-firebase-emulator.sh \
+          ${{ inputs.emulators }} \
+          ${{ inputs.project-id }} \
+          ${{ inputs.max-retries }} \
+          ${{ inputs.max-checks }} \
+          ${{ inputs.wait-time }} \
+          ${{ inputs.check-port }}

